src/uikit.cs

```diff
@@ -293,15 +293,15 @@ partial interface NSFileProviderExtension {
 	    [Static, Export ("writePlaceholderAtURL:withMetadata:error:")]
 	    bool WritePlaceholder (NSUrl placeholderUrl, NSDictionary metadata, ref NSError error);
 	
-		[Deprecated (PlatformName.iOS, 11, 0, message: "Use 'FileProvider.NSFileProviderManager.GetPlaceholderUrl (NSUrl)' instead.")]
+		[Deprecated (PlatformName.iOS, 11, 0, message: "Use 'FileProvider::NSFileProviderManager.GetPlaceholderUrl (NSUrl)' instead.")]
```

Comment:
I _see_ now, the `::` is not the right syntax. Use the full syntax like `global::FileProvider.NSFileProviderManager.GetPlaceholderUrl` to remove any ambiguity.